packages/clay-tooltip/stories/index.tsx

```diff
@@ -64,6 +64,13 @@ storiesOf('Components|ClayTooltip', module)
 					>
 						{'Custom Delay'}
 					</button>
+
+					<div
```

Comment:
Maybe it is better to add another example? At first glance I thought the Tooltip position was broken, but it was only because the `div` is 100% 😅.

Reply:
I actually did that on purpose so that the user can see that it aligns to the node the title is on. Maybe making it a different color would help that?

Reply:
Oh yes that's a great strategy. Yes, adding a soft color would help.